### PR TITLE
*: correct comments mistake.

### PR DESCRIPTION
**Pull request by hueypark**

### CODE REVIEW

File: planner/core/plan.go [hunk 1]
```diff
@@ -29,18 +29,18 @@ import (
...
-	// Get the ID in explain statement
+	// ExplainID returns explain statement ID.
 	ExplainID() string
 	// replaceExprColumns replace all the column reference in the plan's expression node.
 	replaceExprColumns(replace map[string]*expression.Column)
 
 	context() sessionctx.Context
 
-	// property.StatsInfo will return the property.StatsInfo for this plan.
+	// statsInfo will return the property.StatsInfo for this plan.
```
Comment (zz-jason) [inline planner/core/plan.go]:
```suggestion
	// statsInfo returns the statistics information in the logical property for this plan.
```

File: planner/core/plan.go [hunk 2]
```diff
@@ -106,7 +106,7 @@ type LogicalPlan interface {
 	// If the column is not in the schema, returns an error.
 	findColumn(*ast.ColumnName) (*expression.Column, int, error)
 
-	// Get all the children.
+	// Children returns all the children.
```
Comment (zz-jason) [inline planner/core/plan.go]:
```suggestion
	// Children returns all the children LogicalPlans.
```

File: planner/core/plan.go [hunk 3]
```diff
@@ -127,13 +127,13 @@ type PhysicalPlan interface {
 	// ExplainInfo returns operator information to be explained.
 	ExplainInfo() string
 
-	// getChildReqProps gets the required property by child index.
+	// GetChildReqProps gets the required property by child index.
```
Comment (zz-jason) [inline planner/core/plan.go]:
```suggestion
	// GetChildReqProps gets the required physical property property of child by its index.
```

File: planner/core/plan.go [hunk 4]
```diff
@@ -127,13 +127,13 @@ type PhysicalPlan interface {
...
 	GetChildReqProps(idx int) *property.PhysicalProperty
 
 	// StatsCount returns the count of property.StatsInfo for this plan.
 	StatsCount() float64
 
-	// Get all the children.
+	// Children returns all the children.
```
Comment (zz-jason) [inline planner/core/plan.go]:
```suggestion
	// Children returns all the children PhysicalPlans.
```

File: planner/core/point_get_plan.go [hunk 5]
```diff
@@ -98,7 +98,7 @@ func (p *PointGetPlan) StatsCount() float64 {
 	return 1
 }
 
-// StatsCount will return the the RowCount of property.StatsInfo for this plan.
+// statsInfo will return the the RowCount of property.StatsInfo for this plan.
```
Comment (zz-jason) [inline planner/core/point_get_plan.go]:
```suggestion
// statsInfo returns property.StatsInfo.
```

File: executor/aggfuncs/func_group_concat.go [hunk 6]
```diff
@@ -30,9 +30,9 @@ type baseGroupConcat4String struct {
 
 	sep    string
 	maxLen uint64
+	// truncated acts as a sentinel to indicate whether this warning has already been generated.
 	// According to MySQL, a 'group_concat' function generates exactly one 'truncated' warning during its life time, no matter
```
Comment (zimulala) [inline executor/aggfuncs/func_group_concat.go]: Please remove all the `'`.

File: executor/sort.go [hunk 7]
```diff
@@ -47,7 +47,7 @@ type SortExec struct {
 	keyChunks *chunk.List
 	// rowChunks is the chunks to store row values.
 	rowChunks *chunk.List
-	// rowPointer store the chunk index and row index for each row.
+	// rowPtrs store the chunk index and row index for each row.
```
Comment (zimulala) [inline executor/sort.go]:
```suggestion
	// rowPtrs stores the chunk index and row index for each row.
```

File: planner/core/find_best_task.go [hunk 8]
```diff
@@ -309,7 +309,7 @@ func isCoveringIndex(columns []*expression.Column, indexColumns []*model.IndexCo
 	return true
 }
 
-// If there is a table reader which needs to keep order, we should append a pk to table scan.
+// appendExtraHandleCol appends a pk to table scan if there is a table reader which needs to keep order.
```
Comment (zimulala) [inline planner/core/find_best_task.go]: s/`pk`/`PK`

File: planner/core/find_best_task.go [hunk 9]
```diff
@@ -418,6 +418,7 @@ func (ds *DataSource) convertToIndexScan(prop *property.PhysicalProperty, path *
 	return task, nil
 }
 
+// initSchema initialize schema.
```
Comment (zimulala) [inline planner/core/find_best_task.go]: s/`initialize`/`initializes`

File: planner/core/rule_max_min_eliminate.go [hunk 10]
```diff
@@ -30,7 +30,7 @@ func (a *maxMinEliminator) optimize(p LogicalPlan) (LogicalPlan, error) {
 	return p, nil
 }
 
-// Try to convert max/min to Limit+Sort operators.
+// eliminateMaxMin try to convert max/min to Limit+Sort operators.
```
Comment (zimulala) [inline planner/core/rule_max_min_eliminate.go]: s/`try`/`tries`

File: types/time.go [hunk 11]
```diff
@@ -811,7 +813,7 @@ type Duration struct {
 	Fsp int
 }
 
-//Add adds d to d, returns a duration value.
+// Add adds d to d, returns a duration value.
```
Comment (zimulala) [inline types/time.go]:
```suggestion
// Add adds v to d, returns a duration value.
```

File: statistics/feedback.go [hunk 12]
```diff
@@ -991,7 +992,7 @@ func (q *QueryFeedback) logDetailedInfo(h *Handle) {
 	}
 }
 
-// getNewCount adjust the estimated `eqCount` and `rangeCount` according to the real count.
+// getNewCountForIndex adjust the estimated `eqCount` and `rangeCount` according to the real count.
```
Comment (zimulala) [inline statistics/feedback.go]: s/`adjust`/`adjusts`

File: util/stringutil/string_util.go [hunk 13]
```diff
@@ -189,6 +189,7 @@ func CompilePattern(pattern string, escape byte) (patChars, patTypes []byte) {
 
 const caseDiff = 'a' - 'A'
 
+// matchByteCI checks two byte is matched.
```
Comment (eurekaka) [inline util/stringutil/string_util.go]:
```suggestion
// matchByteCI checks if two byte values are matched.
```

File: util/sqlexec/restricted_sql_executor.go [hunk 14]
```diff
@@ -88,7 +88,7 @@ type RecordSet interface {
 	// Next reads records into chunk.
 	Next(ctx context.Context, req *chunk.RecordBatch) error
 
-	//NewRecordBatch create a recordBatch.
+	// NewRecordBatch create a recordBatch.
```
Comment (eurekaka) [inline util/sqlexec/restricted_sql_executor.go]:
```suggestion
	// NewRecordBatch creates a recordBatch.
```

File: util/ranger/ranger_test.go [hunk 15]
```diff
@@ -603,6 +603,7 @@ func (s *testRangerSuite) TestIndexRange(c *C) {
 	}
 }
 
+// TestIndexRangeForUnsignedInt tests index range for unsigned int.
```
Comment (eurekaka) [inline util/ranger/ranger_test.go]:
```suggestion
// TestIndexRangeForUnsignedInt tests index range for unsigned int. (for issue #6661)
```

File: util/ranger/ranger.go [hunk 16]
```diff
@@ -463,6 +463,7 @@ func fixRangeDatum(v *types.Datum, length int, tp *types.FieldType) bool {
 	return false
 }
 
+// newFieldType returns field type.
```
Comment (eurekaka) [inline util/ranger/ranger.go]:
```suggestion
// newFieldType creates a new field type.
```

File: executor/aggfuncs/aggfuncs.go [hunk 17]
```diff
@@ -23,64 +23,64 @@ import (
...
-
-	// All the AggFunc implementations for "BIT_OR" are listed here.
-	_ AggFunc = (*bitOrUint64)(nil)
-
-	// All the AggFunc implementations for "BIT_XOR" are listed here.
-	_ AggFunc = (*bitXorUint64)(nil)
-
-	// All the AggFunc implementations for "BIT_AND" are listed here.
-	_ AggFunc = (*bitAndUint64)(nil)
+// All the AggFunc implementations for "COUNT" are listed here.
```
[the remark this answers is not in the document]
Comment (hueypark) [reply executor/aggfuncs/aggfuncs.go]: goword forces the top-most comment to begin with the name of the field.

We have to do one of three things.
    1. Follow the rules.
    2. Do not use goword.
    3. Send a PR to the goword to modify it.

File: executor/table_reader.go [hunk 18]
```diff
@@ -155,6 +155,7 @@ func (e *TableReaderExecutor) buildResp(ctx context.Context, ranges []*ranger.Ra
 
 type tableResultHandler struct {
 	// If the pk is unsigned and we have KeepOrder=true.
+
```
[the remark this answers is not in the document]
Comment (hueypark) [reply executor/table_reader.go]: See the comment on goword in `aggfuncs.go`

File: distsql/stream.go [hunk 19]
```diff
@@ -35,6 +35,7 @@ type streamResult struct {
 	fieldTypes []*types.FieldType
 	ctx        sessionctx.Context
 
+	// curr represents current chunck of stream.
```
[the remark this answers is not in the document]
Comment (hueypark) [reply distsql/stream.go]: Thank you for review.
Done.

File: domain/global_vars_cache.go [hunk 20]
```diff
@@ -28,6 +28,7 @@ type GlobalVariableCache struct {
 	rows       []chunk.Row
 	fields     []*ast.ResultField
 
+	// disable represents the disable state of GlobalVariableCache.
```
[the remark this answers is not in the document]
Comment (hueypark) [reply domain/global_vars_cache.go]: Done.

File: executor/aggfuncs/func_avg.go [hunk 21]
```diff
@@ -25,8 +25,8 @@ import (
 
 // All the following avg function implementations return the decimal result,
 // which store the partial results in "partialResult4AvgDecimal".
-//
-// "baseAvgDecimal" is wrapped by:
+
+// baseAvgDecimal is wrapped by:
 // - "avgOriginal4Decimal"
```
[the remark this answers is not in the document]
Comment (hueypark) [reply executor/aggfuncs/func_avg.go]: Done.

File: executor/aggregate.go [hunk 22]
```diff
@@ -147,6 +147,7 @@ type HashAggExec struct {
 	groupKeyBuffer   []byte
 	groupValDatums   []types.Datum
 
+	// isUnparallelExec represents unparallel execute.
```
[the remark this answers is not in the document]
Comment (hueypark) [reply executor/aggregate.go]: Done.

File: executor/aggregate.go [hunk 23]
```diff
@@ -601,7 +602,7 @@ func (e *HashAggExec) prepare4ParallelExec(ctx context.Context) {
 	go e.waitFinalWorkerAndCloseFinalOutput(finalWorkerWaitGroup)
 }
 
-// HashAggExec employs one input reader, M partial workers and N final workers to execute parallelly.
+// parallelExec makes HashAggExec employs one input reader, M partial workers and N final workers to execute parallelly.
```
[the remark this answers is not in the document]
Comment (hueypark) [reply executor/aggregate.go]: Done.

File: executor/merge_join.go [hunk 24]
```diff
@@ -73,7 +73,7 @@ type mergeJoinInnerTable struct {
 	joinKeys []*expression.Column
 	ctx      context.Context
 
-	// for chunk executions
+	// sameKeyRows and belows for chunk executions.
```
[the remark this answers is not in the document]
Comment (hueypark) [reply executor/merge_join.go]: Done.

File: executor/write_test.go [hunk 25]
```diff
@@ -1957,6 +1957,7 @@ func (s *testSuite2) TestLoadDataIgnoreLines(c *C) {
 	checkCases(tests, ld, c, tk, ctx, selectSQL, deleteSQL)
 }
 
+// TestLoadDataOverflowBigintUnsigned tests load data overflow big int unsigned.
```
[the remark this answers is not in the document]
Comment (hueypark) [reply executor/write_test.go]: Done.

File: planner/core/exhaust_physical_plans.go [hunk 26]
```diff
@@ -83,7 +83,7 @@ func (p *LogicalJoin) moveEqualToOtherConditions(offsets []int) []expression.Exp
 	return otherConds
 }
 
-// Only if the input required prop is the prefix fo join keys, we can pass through this property.
+// tryToGetChildReqProp does only if the input required prop is the prefix fo join keys, we can pass through this property.
```
[the remark this answers is not in the document]
Comment (hueypark) [reply planner/core/exhaust_physical_plans.go]: Done.

File: planner/core/exhaust_physical_plans.go [hunk 27]
```diff
@@ -151,7 +151,7 @@ func (p *LogicalJoin) getMergeJoin(prop *property.PhysicalProperty) []PhysicalPl
 	return joins
 }
 
-// Change JoinKeys order, by offsets array
+// getNewJoinKeysByOffsets changes JoinKeys order, by offsets array
```
[the remark this answers is not in the document]
Comment (hueypark) [reply planner/core/exhaust_physical_plans.go]: Done.

File: planner/core/exhaust_physical_plans.go [hunk 28]
```diff
@@ -291,6 +291,7 @@ func joinKeysMatchIndex(keys, indexCols []*expression.Column, colLengths []int)
 	return keyOff2IdxOff
 }
 
+// constructIndexJoin construct index join.
```
[the remark this answers is not in the document]
Comment (hueypark) [reply planner/core/exhaust_physical_plans.go]: Done.

File: planner/core/exhaust_physical_plans.go [hunk 29]
```diff
@@ -644,6 +645,7 @@ func (p *LogicalJoin) tryToGetIndexJoin(prop *property.PhysicalProperty) ([]Phys
 	return plans, false
 }
 
+// exhaustPhysicalPlans generates physical plans.
```
[the remark this answers is not in the document]
Comment (hueypark) [reply planner/core/exhaust_physical_plans.go]: Done.

File: planner/core/exhaust_physical_plans.go [hunk 30]
```diff
@@ -734,7 +736,7 @@ func (lt *LogicalTopN) getPhysLimits() []PhysicalPlan {
 	return ret
 }
 
-// Check if this prop's columns can match by items totally.
+// matchItems checks if this prop's columns can match by items totally.
```
[the remark this answers is not in the document]
Comment (hueypark) [reply planner/core/exhaust_physical_plans.go]: Done.

File: planner/core/physical_plans.go [hunk 31]
```diff
@@ -117,6 +117,7 @@ type PhysicalIndexScan struct {
 
 	rangeDecidedBy []*expression.Column
 
+	// isPartition represents partitionability of index scan.
```
[the remark this answers is not in the document]
Comment (hueypark) [reply planner/core/physical_plans.go]: Done.

File: planner/core/plan.go [hunk 32]
```diff
@@ -29,18 +29,18 @@ import (
...
 // finally used by the executor to create a Cursor which executes the statement.
 type Plan interface {
-	// Get the schema.
+	// Schema returns schema.
 	Schema() *expression.Schema
-	// Get the ID.
+	// ID returns ID.
 	ID() int
-	// Get the ID in explain statement
+	// ExplainID returns explain statement ID.
```
[the remark this answers is not in the document]
Comment (hueypark) [reply planner/core/plan.go]: Done.

File: planner/core/plan.go (hunk 1 above)
Comment (hueypark) [reply planner/core/plan.go]: Done.

File: planner/core/plan.go (hunk 2 above)
Comment (hueypark) [reply planner/core/plan.go]: Done.

File: planner/core/plan.go (hunk 4 above)
Comment (hueypark) [reply planner/core/plan.go]: Done.

File: planner/core/point_get_plan.go (hunk 5 above)
Comment (hueypark) [reply planner/core/point_get_plan.go]: Done.

File: executor/aggfuncs/func_group_concat.go (hunk 6 above)
Comment (hueypark) [reply executor/aggfuncs/func_group_concat.go]: Done.

File: executor/sort.go (hunk 7 above)
Comment (hueypark) [reply executor/sort.go]: Done.

File: planner/core/find_best_task.go (hunk 9 above)
Comment (hueypark) [reply planner/core/find_best_task.go]: Done.

File: planner/core/rule_max_min_eliminate.go (hunk 10 above)
Comment (hueypark) [reply planner/core/rule_max_min_eliminate.go]: Done.

File: types/time.go (hunk 11 above)
Comment (hueypark) [reply types/time.go]: Done.

File: statistics/feedback.go (hunk 12 above)
Comment (hueypark) [reply statistics/feedback.go]: Done.

File: util/stringutil/string_util.go (hunk 13 above)
Comment (hueypark) [reply util/stringutil/string_util.go]: Done.

File: util/sqlexec/restricted_sql_executor.go (hunk 14 above)
Comment (hueypark) [reply util/sqlexec/restricted_sql_executor.go]: Done.

File: util/ranger/ranger_test.go (hunk 15 above)
Comment (hueypark) [reply util/ranger/ranger_test.go]: Done.

File: util/ranger/ranger.go (hunk 16 above)
Comment (hueypark) [reply util/ranger/ranger.go]: Done.

File: executor/table_reader.go [hunk 33]
```diff
@@ -155,6 +155,7 @@ func (e *TableReaderExecutor) buildResp(ctx context.Context, ranges []*ranger.Ra
...
 	// optionalResult handles the request whose range is in signed int range.
 	// result handles the request whose range is exceed signed int range.
 	// Otherwise, we just set optionalFinished true and the result handles the whole ranges.
```
[the remark this answers is not in the document]
Comment (hueypark) [reply executor/table_reader.go]: See the comment on goword in `aggfuncs.go`
I think this is the same case.

**Pull request closed without merging**